### PR TITLE
Add --no-exc-wrap to krun

### LINKED ISSUE #1990: [KIP] - `--no-exc-wrap` flag for `krun`
Motivation
----------

Needed for https://github.com/kframework/k/pull/1884#issuecomment-845164891 and https://github.com/kframework/k/pull/1987

Documentation
-------------

Do not wrap exception messages to 80 chars. Keep long lines.

Testing Approach
----------------

Running `krun` on the `issue-1683-cfgVarsWarns` integration test now gives an error spitted on two lines. If given the `--no-exc-wrap` flag, it should print the error on a single line.

### PR DESCRIPTION
**Pull request by radumereuta**
Fixes: #1990

### CODE REVIEW

File: k-distribution/src/main/scripts/bin/krun [hunk 1]
```diff
@@ -70,13 +70,15 @@ execute () {
 
 KX=$(basename "$0")
 
+FoldLines="fold -s"
```
Comment (ehildenb) [inline k-distribution/src/main/scripts/bin/krun]: Only complaint is that `FoldLines` should likely be `fold_lines` for consistency with bash style

File: k-distribution/src/main/scripts/bin/krun [hunk 2]
```diff
@@ -253,6 +257,10 @@ do
       shift
       ;;
 
+      --no-exc-wrap)
+      fold_lines=tee
```
Comment (dwightguth) [inline k-distribution/src/main/scripts/bin/krun]: `cat -` seems more appropriate to me.

File: k-distribution/src/main/scripts/bin/krun [hunk 3]
```diff
@@ -70,13 +70,15 @@ execute () {
...
+
 error () {
-  printf "[Error] Critical: $@\n" | fold -s 1>&2
+  printf "[Error] Critical: $@\n" | $fold_lines 1>&2
   exit $result
 }
 
 warning () {
-  printf "[Warning] Compiler: $@\n" | fold -s 1>&2
+  printf "[Warning] Compiler: $@\n" $fold_lines 1>&2
```
Comment (ehildenb) [inline k-distribution/src/main/scripts/bin/krun]:
```suggestion
  printf "[Warning] Compiler: $@\n" | $fold_lines 1>&2
```

I think you deleted the `| ` which you need?

Comment (radumereuta) [reply k-distribution/src/main/scripts/bin/krun]: Right, I missed that.

**Review by ehildenb — APPROVED** [review]

**Pull request merged**